src/canister/individual_user_template/src/lib.rs

```diff
@@ -36,6 +36,7 @@ use shared_utils::{
         GetUserUtilityTokenTransactionHistoryError, UpdateProfileSetUniqueUsernameError,
     },
 };
+use std::collections::BTreeMap;
```

Comment:
Why do we need this?

Reply:
[migrate_hotornot_user_to_yral.rs](https://github.com/go-bazzinga/hot-or-not-backend-canister/pull/303/files/4d4379108167cd71bd6e422f2663b29d30ae1cc6#diff-b05e5b6b862550c125daf5a61aeb25173d297a5ec5641a83f44cd2cd39416e90) needs BTreeMap. Is there any other file this can be moved? use in migrate_hotornot_user_to_yral file alone doesnt work. export_candid fails.